reliability-engineering/terraform/modules/concourse-worker-pool/iam.tf

```diff
@@ -138,6 +138,17 @@ resource "aws_iam_policy" "concourse_worker_base" {
         "Effect": "Allow",
         "Action": ["ecr:GetAuthorizationToken"],
         "Resource": "*"
+      }, {
+        "Effect": "Allow",
+        "Action": [
+          "logs:CreateLogGroup",
+          "logs:CreateLogStream",
+          "logs:PutLogEvents",
+          "logs:DescribeLogStreams"
+        ],
+        "Resource": [
+          "arn:aws:logs:*:*:*"
```

Comment:
nit: do we want to specify https://www.terraform.io/docs/providers/aws/d/caller_identity.html instead of all accounts